_rules/block-collapsible-cefbef.md

```diff
@@ -45,87 +45,140 @@ _There are no major accessibility support issues known for this rule._
 ### Passed
 
 #### Passed Example 1
-
+The link to skip repeated content is visible, is included in the accessibility tree, and links to a valid HTML element.

```

Comment:
This looks like examples for another rule.

This rule is about toggling visibility of the block of repeated content and is not concerned with the links to skip it.
These are examples for _rules/first-focusable-element-is-link-to-main-8a213c.md

Reply:
ok I misread the applicability. I have not seen this implementation before. Is the concept to have a button above repeated content and toggle display:none? In my experience bypass blocks are used to move the focus beyond repeated content not actually hide navigational elements from the DOM.

Reply:
Ah I see what I did. the hierarchy of rule and subrule wasn't clear to me at first and obviously edited the incorrect file. I think I get this now. Apologies!

Reply:
> ok I misread the expectations. I have not seen this implementation before. Is the concept to have a button above repeated content and toggle display:none? In my experience bypass blocks are used to move the focus beyond repeated content not actually hide navigational elements from the DOM.

Yes. This corresponds to WCAG technique SCR28: https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR28
(see the technique description for a couple of examples)

The idea is that if you can make the repeated content disappear, then you don't need to skip it.